Makefile

```diff
@@ -92,3 +95,47 @@ delete-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+# Create CCS (BYOC) namespace
+.PHONY: create-ccs-namespace
+create-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc apply -f -
+
+# Delete CCS (BYOC) namespace
+.PHONY: delete-ccs-namespace
+delete-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc delete -f -
+
+# Create CCS (BYOC) Secret
+.PHONY: create-ccs-secret
+create-ccs-secret:
+	# Create CCS Secret
+	./hack/scripts/aws/rotate_iam_access_keys.sh -p osd-staging-2 -u osdCcsAdmin -a ${OSD_STAGING_2_AWS_ACCOUNT_ID} -o ccs-secret.yaml
+	@oc apply -f ccs-secret.yaml
+	# Wait for AWS to propogate IAM credentials
+	sleep ${SLEEP_INTERVAL}
+
+# Delete CCS (BYOC) Secret
+.PHONY: delete-ccs-secret
+delete-ccs-secret:
+	# Create CCS Secret
+	@oc delete -f ccs-secret.yaml
+	@rm ccs-secret.yaml
+
+.PHONY: create-ccs-accountclaim
+create-ccs-accountclaim:
+	# Create ccs accountclaim
+	@oc process -p CCS_ACCOUNT_ID=${OSD_STAGING_2_AWS_ACCOUNT_ID} -p NAME=${CCS_CLAIM_NAME} -p NAMESPACE=${CCS_NAMESPACE_NAME} -f hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl | oc apply -f -
```

Comment:
I don't think the Account Claim CR template made it into your commit.

```
error: failed to read input object (not a Template?): the path "hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl" does not exist
```

Reply:
:man_facepalming: thanks.